servers/rendering/renderer_rd/storage_rd/render_scene_data_rd.h

```diff
@@ -126,26 +126,31 @@ class RenderSceneDataRD {
 
 		// Fog
 		uint32_t fog_enabled;
+		uint32_t fog_mode;
 		float fog_density;
 		float fog_height;
 		float fog_height_density;
 
+		float fog_depth_curve;
+		float pad;
+		float fog_depth_begin;
+
```

Comment:
```suggestion
		uint32_t fog_mode;
		float fog_density;
		float fog_height;

		float fog_height_density;
		float fog_depth_curve;
		float pad;
		float fog_depth_begin;

```
Should be grouped into four